tests/test_e2e_integration.py

```diff
@@ -0,0 +1,241 @@
+# Copyright 2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+End-to-end integration tests between the local machine and the remote Orquestra
+platform.
+
+These test cases connect to the Orquestra platform. Prior authentication to the
+Orquestra platform is required for running them. The roundtrip time can be 1-2
+minutes for test cases including assertions on the output of the computation.
+"""
+import pytest
+import numpy as np
+import math
+import yaml
+import os
+
+import pennylane as qml
+from pennylane_orquestra import OrquestraDevice, QeQiskitDevice, QeIBMQDevice
+import pennylane_orquestra.gen_workflow as gw
+from pennylane_orquestra.cli_actions import qe_submit, workflow_details
+
+from conftest import (
+    qe_list_workflow,
+    backend_specs_default,
+    operator_string_default,
+    qasm_circuit_default,
+    resources_default,
+)
+
+qiskit_analytic_specs = '{"module_name": "qeqiskit.simulator", "function_name": "QiskitSimulator", "device_name": "qasm_simulator"}'
+qiskit_sampler_specs = '{"module_name": "qeqiskit.simulator", "function_name": "QiskitSimulator", "device_name": "qasm_simulator", "n_samples": 1000}'
+
+analytic_tol = 10e-10
+
+# The tolerance for sampling is expected to be higher
+tol = 10e-2
+
+# Expected to fail if qe is unavailable (for example for the CI)
+@pytest.mark.xfail(raises=FileNotFoundError)
```

Comment:
Is this because qe is not installed on the CI?

Reply:
Yes, although on second thought it might be overkill as the CI skips it explicitly. Will rather remove these, so that if the `e2e` tests are run explicitly, they are not unexpectedly passing (`X`).